src/__tests__/steps/index.test.ts

```diff
@@ -154,7 +160,7 @@ describe('service entities/relationships', () => {
     expect(serviceUserRelationships).toEqual(
       expect.arrayContaining([
         expect.objectContaining({
-          _key: expect.stringMatching(/user:PU.*\|monitors\|service:PS.*/g),
+          // _key: expect.stringMatching(/user:PU.*\|monitors\|service:PS.*/g),
```

Comment:
Did the `_key`s produced change?

Reply:
UGH.  Good catch.  That was not an intended change.  I'll get it uncommented in the next commit.